java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/config/domain/BuildContext.java

```diff
@@ -145,13 +146,17 @@ private void writeDockerFile() throws IOException {
     Path dockerFilePath = workspaceDir.resolve(DOCKERFILE_NAME);
 
     // fail loudly if a Dockerfile already exists
-    if (Files.exists(dockerFilePath)
-        || Files.exists(workspaceDir.resolve("src/main/docker/" + DOCKERFILE_NAME))) {
+    if (Files.exists(dockerFilePath)) {
```

Comment:
I'm thinking we can just have an immutable list of Dockerfile locations and loop through it here. The exception message can simply substitute the location in which it found the unexpected Dockerfile. WDYT?

Reply:
Yup. However, should the case of just the file name be special (that one telling the user he can use 
"custom" option), while the other cases inside various directories just tell the user to remove the file?